CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ Categories for each release: Added, Changed, Deprecated, Removed, Fixed, Securit
 
 ## Unreleased
 
+### Removed
```

Comment:
can you plase move the section below Fixed to unify with previous entries?

Reply:
done (categories were in that order at the top, though).